packages/bindgen/src/templates/jsi.ts

```diff
@@ -287,7 +287,7 @@ function convertPrimToJsi(addon: JsiAddon, type: string, expr: string): string {
     case "std::exception_ptr":
       return `toJsiException(_env, ${expr})`;
     case "std::error_code":
-      return `toJsiErrorCode(_env, ${expr})`;
+      return `toJsiException(_env, ${expr})`;
```

Comment:
???

I would be surprised if this actually compiles.

Reply:
typo sorry